iidm/iidm-serde/src/test/java/com/powsybl/iidm/serde/NetworkSerDeTest.java

```diff
@@ -116,6 +117,21 @@ void testGzipGunzip() throws IOException {
         assertArrayEquals(Files.readAllBytes(file1), Files.readAllBytes(file2));
     }
 
+    @Test
+    void testCopyFormat() throws IOException {
+        Network network = createEurostagTutorialExample1();
+        Path file1 = tmpDir.resolve("n.xml");
+        NetworkSerDe.write(network, file1);
+        Network network2 = NetworkSerDe.copy(network);
+        Path file2 = tmpDir.resolve("n2.xml");
+        NetworkSerDe.write(network2, file2);
+        assertArrayEquals(Files.readAllBytes(file1), Files.readAllBytes(file2));
```

Comment:
The mismatch would be more understandable if you use `ComparisonUtils.assertTxtEquals` instead.